dcr/scenario_utils/check_waagent_log.py

```diff
@@ -96,7 +96,7 @@ def check_waagent_log_for_errors(waagent_log=AGENT_LOG_FILE, ignore=None):
         # 2021-12-20T07:46:23.020197Z INFO ExtHandler ExtHandler [CGW] The agent's process is not within a memory cgroup
         {
             'message': r"The agent's process is not within a memory cgroup",
-            'if': lambda log_line: re.match(r"((centos7\.8)|(redhat7\.8)|(redhat8\.2))\D*", distro,
+            'if': lambda log_line: re.match(r"((centos7\.8)|(centos7\.9)|(redhat7\.8)|(redhat8\.2))\D*", distro,
```

Comment:
Thanks for adding this. I just checked the DCR v2 runs, and we also have Redhat 7.7 in there (though that's not failing). Not sure if that needs to be added to the list here too or not - 

> WALinuxAgent-2.2.38 running on redhat 7.7

Reply:
Cgroups enabled after 7.8. So, 7.7 should not produce this warning